src/Features/LanguageServer/Microsoft.CodeAnalysis.LanguageServer.Xaml/XamlRequestTelemetryLoggerFactory.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Composition;
+using Microsoft.CodeAnalysis.Host.Mef;
+using Microsoft.CodeAnalysis.LanguageServer.Handler;
+
+namespace Microsoft.CodeAnalysis.LanguageServer.Xaml;
+
+[ExportLspServiceFactory(typeof(RequestTelemetryLogger), StringConstants.XamlLspLanguagesContract), Shared]
+internal class XamlRequestTelemetryLoggerFactory : RequestTelemetryLoggerFactory
```

Comment:
this also should not be necessary, the main roslyn server already has this service

Reply:
You're right, I just verified it's not needed at all.

Reply:
Gone